sql/ws-tokens.sql

```diff
@@ -0,0 +1,9 @@
+USE `LXD_Manager`;
+
+CREATE TABLE `ws_tokens` (
+    `id` INT(11) NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    `created_at` DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    `token` TEXT NOT NULL,
+    `used` DATETIME,
+    `user_id` INT(11)
+);
```

Comment:
To be consistent in the column naming convention please can the schema be updated to this 

```suggestion
CREATE TABLE `User_Api_Tokens` (
    `UAT_ID` INT(11) NOT NULL AUTO_INCREMENT PRIMARY KEY,
    `UAT_Created_At` DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
    `UAT_User_ID` INT(11)
    `UAT_Token` TEXT NOT NULL,
    `UAT_Last_Used` DATETIME,
    `UAT_Permanent` TINYINT NOT NULL DEFAULT 0
);
```

Reply:
Done. I used Laravel style column names so that it'll be more database portable and easier to type. Just in case you get around to supporting postgresql. That's because in postgresql, column names with upper case need to be enclosed with double quotes.